lifecycle-operator/controllers/lifecycle/keptntask/job_utils.go

```diff
@@ -138,3 +144,10 @@ func (r *KeptnTaskReconciler) generateJob(ctx context.Context, task *klcv1alpha3
 
 	return job, nil
 }
+
+func getServiceAccount(serviceAccount *klcv1alpha3.ServiceAccountSpec) (string, error) {
+	if serviceAccount == nil {
+		return "", errors.New("ServiceAccount is missing")
```

Comment:
service account should not be a mandatory thing IMO, what I would do in this case is log as info that no service account is configured and not set the service account / return an empty string